package.json

```
@@ -109,6 +109,9 @@
           "items": {
             "type": "string"
           },
+          "default": [
+            "-Xmx512m"
```

```suggestion
            "-Xmx1024m"
```
is what I found optimal, so let's maybe not set it too low?

---

Tried `512m` additionally on akka/akka and apache/spark.
The real memory usage is about `200m` so I think `512` fits better for default.

---

512 should be enough for even large repos unless there have been added new indexes recently that hold onto a lot of memory.